### DISCUSSION TITLE
Do I need to disable any security authentication check on Databricks Account to get HTTP GET requests working?

### DISCUSSION THREAD

**Discussion by heavy-metal-blues-code** [mick2004/AzureAADDatabricksConnector#3] [Q&A] [answered]
Hello @mick2004 and @AbhiDatabricks,
Thank you for sharing the `nested-and-scim-connector` code publication.
I encountered an issue while making an HTTP request at this [line](https://github.com/mick2004/nested-aad-scim-connector/blob/570c67b7142982387923572ca5b9aa58c418d34a/nestedaaddb/databricks_client.py#L28) in JSON format.
The error message says "You need to enable JavaScript to run this app".
I tested the Databricks Account I'm using as a playground via Terraform, and I was able to fetch the requests by only providing the Databricks Account ID Token, so I believe the issue I'm facing while performing the `HTTP` `GET` (`response = requests.get(api_url, headers=my_headers).text`) request might be a mechanism from Databricks Account application to prevent bots. (bot-defeat secure code)
Do you have any idea what the issue could be about?
Is there any secure verification on Databricks Account that has to be enabled/disabled?

Regards,
@heavy-metal-blues-code

**Comment by AbhiDatabricks** [comment, marked as answer]
Hi @heavy-metal-blues-code ,Thank for your comment.I think the Account SCIM URL you are entering is wrong( I saw it last time when someone missed to put "/api/2.0" in URI).Could you double-check it from Account Console? Generally, it's of the format below:

https://accounts.cloud.databricks.com/api/2.0/accounts/<<Account Identifier>>/scim/v2
or
https://accounts.azuredatabricks.net/api/2.0/accounts/<<Account Identifier>>/scim/v2


You can get it via https://learn.microsoft.com/en-us/azure/databricks/administration-guide/users-groups/scim/aad#step-1-configure-azure-databricks

**Reply by heavy-metal-blues-code** [reply]
Whoa! Issue solved!
It was directly to the point @AbhiDatabricks, thanks a bunch.

Regards,
@heavy-metal-blues-code

**Reply by AbhiDatabricks** [reply]
Great ! Thanks for testing out.
